pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -51,6 +52,15 @@ var noDescription string = `# Name: One command of each type
 oc foo
 `
 
+var addOnWithMultilineDescription string = `# Name: foo
+# Description: This is one line
+#   This is second line
```

Comment:
Do we need the indentation? You are suggesting that for multiline descriptionm the successive lines needs to start with 3 spaces (or is it a tab)? 

I was wondering whether it would be easier (for the user) and also nicer to read, if no indentation would be required. 

     # Description: This is one line
     # This is second line
     # This is the third line
     # Required-Vars: This is a new meta header

During parsing of the header you start a new meta tag when you encounter the first 'XYZ:'. Then you aggregate all following lines into the same header until you hit the next tag 'XYA'.

The potential catch is that without indentation you might have issues with lines containing the ':' itself. But this problem exists already. It would probably be a good idea to harden the regexp '(`^# ?(.*):(.*)`)' a bit.